test/analyzeCode.test.ts

```diff
@@ -78,8 +82,8 @@ test("Edit variable", () => {
 });
 
 test("PrefixUnaryExpression and PrefixUnaryExpression", () => {
-    const code = `const f = -5;
-                const w = +6;
+    const code = `var f = -5;
```

Comment:
Generally, in JavaScript it's preferable to use `let` instead of `var`. `var` has scoping rules you wouldn't expect.

[MDN var](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var)

[MDN let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)

Reply:
Good resources. Resolved the issue!